addOns/grpc/src/main/javahelp/org/zaproxy/addon/grpc/resources/help/contents/grpc.html

```diff
@@ -6,8 +6,52 @@
     </TITLE>
 </HEAD>
 <BODY>
-<H1>gRPC Support</H1>
+<h1>gRPC Support</h1>
 This add-on will allow you to inspect, attack gRPC endpoints, and decode protobuf messages. It is currently under development.
 
+<p>gRPC APIs have become increasingly popular for building efficient and robust microservices architectures. However, testing these APIs effectively can be challenging, especially when dealing with Protobuf message encoding and decoding. In this blog post, we&#39;ll explore a custom extension designed to streamline the process of testing gRPC endpoints and handling Protobuf messages.</p>
```

Comment:
I don't think this paragraph should be in the help, in any case the "In this blog post" needs to be changed.

Reply:
should i remove this whole paragraph ?

Reply:
IMO yes, @psiinon (and all) wdyt?

Reply:
Yup I'd agree with that

Reply:
ok i have removed the whole paragraph